akka-http/src/main/java/akka/http/javadsl/coding/Coder.java

```diff
@@ -25,9 +29,40 @@
         this.underlying = underlying;
     }
 
+    public HttpResponse encodeMessage(HttpResponse message) {
```

Comment:
Cool, I wasn't sure to add this method, then I can simplify the signature of the Scala's `encodeMessage` and `decodeMessage` to just take `HttpMessage` (instead of `HttpMessage with MessageTransformers`).
Then, I also question if the `transformEntityDataBytes` should be in `MessageTransformers` at all.